src/api/exercises.js

```diff
@@ -2,21 +2,33 @@ import { Zeeguu_API } from "./classDef";
 import qs from "qs";
 
 Zeeguu_API.prototype.getUserBookmarksToStudy = function (count, callback) {
-  this._getJSON(`bookmarks_to_study/${count}`, callback);
+  this._getJSON(`scheduled_bookmarks_to_study/${count}`, callback);
 };
 
 Zeeguu_API.prototype.getUserBookmarksInPipeline = function (callback) {
   this._getJSON(`bookmarks_in_pipeline`, callback);
 };
 
+Zeeguu_API.prototype.getBookmarksToLearn = function (callback) {
+  this._getJSON(`bookmarks_to_learn_not_scheduled`, callback);
```

Comment:
also, maybe we should try to unify the terminology and use only one of the `learn` or `study`? The first instinct would be to say, let's go with `study`. Or `practice` even?